### DISCUSSION TITLE
default-value field missing in custom-field plugin (in Demo site)

### DISCUSSION THREAD

**Discussion by fabio-pif** [kimai/kimai#4391] [Questions & Answers]
Hello,
i'm trying Kimai with the demo site, but i've an issue when i would like to define a new custom field as Drop-down list: the default-value field is missing and i can't enter the lists of value as suggeted in thd documentation (i.e. Banana,Apple,Orange,Pineapple,Peach ). See attached screenshot.
![Clipboard01](https://github.com/kimai/kimai/assets/149665621/300cb429-39af-4acd-9ffc-39e923c90580)

It's a bug ? how can be solved?

Thank's for help
F.

**Comment by kevinpapst** [comment]
Default value can be added when re-opening the created field.
